src/vis/vis.js

```diff
@@ -624,20 +624,36 @@ var Vis = View.extend({
   // public methods
   //
 
-  // get the native map used behind the scenes
+  /**
+   * @return the native map used behind the scenes {L.Map} or {google.maps.Map}
+   */
   getNativeMap: function () {
     return this.mapView.getNativeMap();
   },
 
   // returns an array of layers
-  // TODO: Rename to getLayerViews
-  getLayers: function () {
```

Comment:
This method is being used [by the _setLayerOptions method](https://github.com/CartoDB/cartodb.js/blob/api/src/vis/vis.js#L66), which is not being used. I think we should remove `setLayerOptions` since no tests fail for that reason.

That method is also being used [here](https://github.com/CartoDB/cartodb.js/blob/v4/src/vis/overlays.js#L297), [here](https://github.com/CartoDB/cartodb.js/blob/v4/src/vis/overlays.js#L300), and [here](https://github.com/CartoDB/cartodb.js/blob/v4/src/vis/overlays.js#L308). We can just rename these usages to getLayerViews for now.